deptofdefense/solo
# Issue#18 Create Hello World Front-end React App

Thanks for submitting a pull request! Below are a few things you can do to help us more quickly review your changes.

#18 Create Hello World Front-end React App
#12 Created using prettier for now

## Checklist
I have…

## Summary of Changes
This pull request…
- Used create-react-app to bootstrap a new typescript react application
- Set up typechecking, linting, and prettier
- Replaced CRA starter with a single "hello world" component
- Wrote 3 unit tests

## Testing
To verify the changes proposed in this pull request…
1. From the frontend/ folder
2. install dependencies (```npm i```)
3. Run unit tests (```npm run test```)
4. Lint (```npm run lint```)
5. Start dev server (```npm run start```)


## Screenshots

<img width="752" alt="Screen Shot 2020-01-17 at 12 09 51 PM" src="https://user-images.githubusercontent.com/59582489/72631526-5a026880-3922-11ea-963e-d9aba6815533.png">
